test/error.test.js

```
@@ -115,4 +115,18 @@ describe("Test error", async function () {
+
+    it("should test shouldThrow", async function () {
+        const error = { code: -32003 };
+        assert(shouldThrow(error));
+
+        error.code = 4001;
+        assert(shouldThrow(error));
+
+        error.code = 5000;
+        assert(shouldThrow(error));
+
+        delete error.code;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)